docs/src/components/layouts/utility-page-layout/utility-class-tables/token-demonstration.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import PropType from 'prop-types';
+
+/**
+ * @typedef TokenDemonstrationClass
+ * @param {string} token_name The class name.
+ * @param {string} token_default The class name.
+ * @param {string} settings_variable The class name.
+ * @param {string} hex_value The calculated value of the size of the token.
+ * @param {string} additional_example_classes Any additional classes to add to the example.
+
+ */
+
+/**
+ * Helper to draw a utility class example.
+ * @param {TokenDemonstrationClass} utilityToken The utility class.
+ * @returns
+ */
+const drawRow = (utilityToken, utilityTokenDisplayParams) => {
```

Comment:
Copied in un-resolved comment that somehow GitHub was not smart about.

Oh, also this should not be in utility-class-tables.

Once again, I have no concerns with making token demonstration be consistent, but there are some issues here:

A token is not a utility, so the class names and data structure are confusing.
A token does not have a class name.
There are different types of tokens. So this really looks like a color tokens table. The font family or spacing tokens take on different displays. So if this is to be kept, and again, with tweaks makes sense, it should probably be something like ColorTokenDemonstration.
Also, in a further enhancement/expectation. The color hex values should be looked up from a json file. The hex values for utility documentation should look up the same hex values. Once we have multiple places for calculated values to be displayed, we have multiple places that can be wrong. I am not asking for this, I ran out of time, but am demonstrating where the thinking should be for tokens and utility interactions.

We should also hyperlink all system tokens to the space where they are documented like USWDS does. Once again, this needs the framework in place.

Reply:
This all ended up getting resolved as a separate design_tokens element in the page.